src/helpers/dateFormatter.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/prefer-default-export
+export const standardFormatter = (date) => {
```

Comment:
what about naming it `formatDate`: it's a verb (that is good for a function) and makes its purpose easier to understand